app/src/main/java/com/android/sample/ui/authentication/SignUp.kt

```diff
@@ -214,14 +189,7 @@ fun SignUpScreen(navigationActions: NavigationActions) {
                     modifier =
                         Modifier.testTag("loginClickableText")
                             .clickable(onClick = { navigationActions.navigateTo(Screen.LOGIN) }),
-                    style =
-                        TextStyle(
-                            fontSize = 14.sp,
-                            fontFamily = FontFamily(Font(R.font.roboto)),
-                            fontWeight = FontWeight(500),
-                            letterSpacing = 0.14.sp,
-                            brush = PrimaryGradientBrush,
-                            textDecoration = TextDecoration.Underline))
+                    style = TypographyBeatLink.labelMedium)
```

Comment:
Consider utilizing MaterialTheme.typography to further centralize theme control and maintain consistency throughout the application.

```suggestion
                    style = MaterialTheme.typography.labelMedium)
```